lib/ome/common/base64.h

```diff
@@ -0,0 +1,323 @@
+/*
+ * #%L
+ * OME-COMMON C++ library for C++ compatibility/portability
+ * %%
+ * Copyright © 2006 - 2015 Open Microscopy Environment:
+ *   - Massachusetts Institute of Technology
+ *   - National Institutes of Health
```

Comment:
Originally I copied the Java boilerplate.  Since I wrote pretty much all of the C++ code, we could perhaps remove them if we wanted.  I think the remaining contributions are all Dundee-related as well, maybe there's a one-liner from an external contributor somewhere.